about.html

```diff
@@ -0,0 +1,212 @@
+                        <a href="index.html#program" class="navbar-link">Program</a>
+                    </li>
+                    <li class="navbar-item">
+                        <a href="index.html#speakersSection" class="navbar-link">Chefs</a>
+                    </li>
+                    <li class="navbar-item">
+                        <a href="#partners" class="navbar-link">Partners</a>
+                    </li>
+                    <li class="navbar-item">
+                        <a href="#footer" class="navbar-link">Contact</a>
+                    </li>
+                    <li class="navbar-item ticket-item">
+                        <a href="" class="navbar-link ticket-link">Buy tickets</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+    </section>
+    <!--End-->
+
+    <!--Main Section-->
+    <main class="main__section">
+        <section class="main__container">
+            <h1 class="main__message">"Hello! Sharing World!"</h1>
+            <div class="main__title__container">
+                <h2 class="main__title__one">CREATIVE COMMONS</h2>
+                <h2 class="main__title__two">GLOBAL SUMMIT 2015</h2>
+                <span class="bottom-line-desktop"></span>
+            </div>
+            <article class="main__features">
+                <p class="main__paragraph__mobile">
+                    The Global Summit brings together the community of
+                    experts, academics, and activists who comprise the Creative Commons affiliate network in a different
+                    country every two years.
+                    This year, we hope to expand our invitation list including organizations and individuals.
+                </p>
+                <p class="main__paragraph__desktop">
+                    The Global Summit brings together the community of experts, academics, and activists who comprise
+                    the Creative Commons affiliate network in a different country every two years.
+                    This year, we hope to expand our invitation list including organizations and individuals who want to
+                    work with us on shared projects that advance the cause of the Commons, free culture and open
+                    knowledge.
+                    So if you're active and engaged in the worlds of open content and knowledge - free software
+                    advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, governments and
+                    foundations, lawyers, and activists - we hope you'll consider joining us this year to build a
+                    stronger, more vibrant commons together
+                </p>
+                <h2 class="main__date">Please contact us per Email for nay further questions about CC Global Summit
+                    2015!</h2>
+                <h3 class="main__subtitle">ccsummit2015@cckorea.org</h3>
+            </article>
+        </section>
```

Comment:
- [ ] This section has a background image on the provided design. Kindly stick to the design as much as possible to follow the project requirement.
### Your design :point_down: 
![Screenshot from 2023-07-08 11-57-31](https://github.com/CVILLA09/capstone-project/assets/91142432/8e7e4074-a2a6-46c5-9eb8-acba6b7cbe3a)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-07-08 11-57-55](https://github.com/CVILLA09/capstone-project/assets/91142432/ca6f44fc-63d1-47ed-901f-e7405a5ecb7f)